docs/runbook.rst

```diff
@@ -41,13 +41,29 @@ Update redis by running:
 
 ::
 
-    rc update -p project -e environment -f feature_flag -s state
+    rc update-redis -p project -e environment -f feature_flag -s state
```

Comment:
I wonder if it would be better to made this an argument instead of a new command. This way if we decided to support additional data stores in the future we can do that. 

So I would imagine adding a new flag called `-d` for datastore (or something like that). This would make the command look like this: 

```
rc update -d dynamodb -p project -e environment -f feature_flag -s state
```

Reply:
Sounds good, will make the change